pom.xml

```diff
@@ -27,10 +27,10 @@
   <packaging>jar</packaging>
 
   <properties>
-    <cdap.version>6.6.0</cdap.version>
+    <cdap.version>6.8.0</cdap.version>
```

Comment:
does this need to be bumped? Seems like it shouldn't be needed if we're just changing dependencies.

Note that any time this is bumped, the lower bound of the parent versions also needs to be bumped, limiting which versions of CDAP that the plugin can be used with.

Reply:
this isn't needed indeed. Reverted this change.